File: Beginner/Bhaskara's_Formula_1036/Main.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String[] inputtedValues = sc.nextLine().split(" ");
        double a = Double.parseDouble(inputtedValues[0]);
        double b = Double.parseDouble(inputtedValues[1]);
        double c = Double.parseDouble(inputtedValues[2]);

        double delta = Math.pow(b, 2) - (4 * a * c);
        if ((a == 0) || (delta < 0)) {
            System.out.println("Impossivel calcular");
        } else {
            double x1 = (-b + Math.sqrt(delta)) / (2 * a);
            double x2 = (-b - Math.sqrt(delta)) / (2 * a);
            System.out.printf("R1 = %.5f\n", x1);
            System.out.printf("R2 = %.5f\n", x2);
        }
        sc.close();
    }
}

File: Beginner/Bhaskara's_Formula_1036/main.py
import math

input_values = list(map(float, input().split(" ")))
a, b, c = input_values[0], input_values[1], input_values[2]

delta = math.pow(b, 2) - (4 * a * c)

if a == 0 or delta < 0:
    print('Impossivel calcular')
else:
    x1 = (-b + math.sqrt(delta)) / (2 * a)
    x2 = (-b - math.sqrt(delta)) / (2 * a)
    print(f'R1 = {x1:.5f}')
    print(f'R2 = {x2:.5f}')
